README.rst

```diff
@@ -24,8 +24,8 @@ web3j: Web3 Java Ethereum Ðapp API
    :target: https://gitter.im/web3j/web3j?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
    :alt: Join the chat at https://gitter.im/web3j/web3j
 
-web3j is a lightweight, reactive, type safe Java and Android library for integrating with clients (nodes) on
```

Comment:
I tried to make sure that both READMEs (master and android) were consistent, but might have inadvertently broken some comments... please double-check!

Reply:
Also, wasn't sure if this block has the correct versions of dependencies:
```
Java 8:

.. code-block:: xml

   <dependency>
     <groupId>org.web3j</groupId>
     <artifactId>core</artifactId>
     <version>2.3.1</version>
   </dependency>

Android:

.. code-block:: xml

   <dependency>
     <groupId>org.web3j</groupId>
     <artifactId>core-android</artifactId>
     <version>2.2.1</version>
   </dependency>
```

Reply:
That will be because I hadn't pulled the documentation updates since releasing 3.0.1-android. Normally I don't update the README.md until I've done the release to Bintray & Maven.